importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -2472,7 +2472,10 @@ void MusicXMLParserDirection::direction(const QString& partId,
                   else {
                         _tpoSound /= 60;
                         t = new TempoText(_score);
-                        t->setXmlText(_wordsText + _metroText);
+                        QString rawWordsText = _wordsText;
+                        rawWordsText.remove(QRegularExpression("(<.*?>)"));
+                        QString sep = _metroText != "" && _wordsText != "" && rawWordsText.back() != ' ' ? " " : "";
```

Comment:
`QString::back()` is not available in Qt 5.9, which is what the 3.x branch is using, it got introduced in Qt 5.10, see https://doc.qt.io/qt-5/qstring.html#back.
Using `rawWordsText.at(rawWordsText.size() - 1)` instead should fix this.

Reply:
Actually maybe this:

    #if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
                        QString sep = _metroText != "" && _wordsText != "" && rawWordsText.back() != ' ' ? " " : "";
    #else
                        QString sep = _metroText != "" && _wordsText != "" && rawWordsText.at(rawWordsText.size() - 1) != ' ' ? " " : "";
    #endif